src/finch/symbolic/__init__.py

```diff
@@ -1,14 +1,23 @@
-from .rewriters import *
-from .term import *
-from .gensym import *
+from .gensym import gensym
+from .rewriters import (
+    Chain,
+    PostWalk,
+    PreWalk,
+    Rewrite,
+)
+from .term import (
+    PostOrderDFS,
+    PreOrderDFS,
+    Term,
+)
 
 __all__ = [
     "PostOrderDFS",
     "PreOrderDFS",
     "Term",
-    "Rewriter",
+    "Rewrite",
```

Comment:
Typo maybe? There's no symbol named `Rewriter` that I can find in the library.

Reply:
That's right.